public/js/helpers/add_day.js

```diff
@@ -0,0 +1,8 @@
+var addDay = function (dataSet) {
```

Comment:
function should take date and number of days and return the date which is the given number of days after the given date:
e.g. 
```js
addDays('2015-12-30', 2) // should return '2016-01-01'
```

Reply:
I'm not sure I want to do this unless if it's what the product owner wants. cause I personally prefer so when you click on a day you get all the events **in that day only**.
So what I would suggest is open an issue to Silke ( as mentioned [here](https://github.com/foundersandcoders/nazareth-events/pull/21#discussion_r138018078) )

Reply:
regardless, function should definitely take `date` as a parameter and not `dataset`